camayoc/types/settings.py

```diff
@@ -24,6 +24,7 @@ class QuipucordsServerOptions(BaseModel):
 
 class QuipucordsCLIOptions(BaseModel):
     executable: Optional[str] = "qpc"
+    display_name: Optional[str] = "qpc"
```

Comment:
I think there should also be a validator in https://github.com/quipucords/camayoc/blob/master/camayoc/config.py#L28

Reply:
updated!